app/Http/Controllers/Api/UserController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers\Api;
```

Comment:
Technically I should this should be under `App\Domains\Settings\ManageUsers\Api` instead